pyrsistent/_pset.py

```diff
@@ -1,8 +1,6 @@
-from ._compat import Set, Hashable
+from collections.abc import Set, Hashable
 import sys
-from pyrsistent._pmap import pmap
-
-PY2 = sys.version_info[0] < 3
+from pyrsistent._pmap import pma
```

Comment:
This import is broken (should be "import pmap"). This causes a lot of tests to fail.

Reply:
Done